site/content/docs/11-deploying.md

```diff
@@ -61,4 +61,72 @@ When deploying to [Now][], you can pass the environment variable into Now itself
 now -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
-[Now]: https://zeit.co/now
\ No newline at end of file
+[Now]: https://zeit.co/now
+
+
+## Deploying with Docker
+
+[Docker](https://docs.docker.com/install/) can be used to build an image and run it on your local system. Doing this locally allows testing the build process before building and deploying it on a cloud platform. The Docker image can be build by adding a `Dockerfile` into your project:
+
+```Dockerfile
+# Dockerfile
+FROM mhart/alpine-node:12 as build
+
+WORKDIR /app
+COPY . .
+RUN npm ci
+RUN npm run build
+
+FROM mhart/alpine-node:12 as prod
+
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci --production
+
+FROM mhart/alpine-node:slim-12
+
+WORKDIR /app
+COPY --from=build /app/__sapper__/build __sapper__/build
+COPY --from=build /app/static static
+COPY --from=prod /app .
+
+CMD ["node", "__sapper__/build"]
+```
+
+This Dockerfile is a [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) that performs three steps:
+
+- Perform a Sapper build with development packages
```

Comment:
What do you mean by "development packages"?  I wouldn't expect to use development packages in a production build

Reply:
So you can run your tests in CI. If we don't have the dev stuff installed, we can't do that and it is common practice.

Reply:
Did an attempt to be more comprehensible... the point that i tried to make was the steps will result in an image that doesn't contain the sapper/svelte compiler itself. Hopefully the text is now more explicit...